src/containers/Exhibitions/index.jsx

```diff
@@ -60,9 +60,9 @@ const Exhibitions = () => {
       <div className={styles.cardsContainer} loading>
         {loading
           ? arraySkeleton.map((e) => (
-              <Skeleton key={e.id} animation="wave" variant="rectangle" className={styles.skeleton}>
-                <ExhibitionCard />
-              </Skeleton>
+              <Box margin="20px">
```

Comment:
Since you're rendering a list of boxes, you should add a `key` prop to each instance of those.